config/gitleaks.toml

```diff
@@ -489,9 +489,6 @@ keywords = [
     "key","api","token","secret","client","passwd","password","auth","access",
 ]
 [rules.allowlist]
-paths = [
-  '''Database.refactorlog'''
-]
```

Comment:
@jankonikola93 Was this deletion intentional? Not sure why this was here in the first place to be honest, but it seems unrelated to your PR so we should probably leave it as-is unless there is some reason to remove it...?

Reply:
It was added when I run script for converting rules to gitleaks.toml file. I was not sure also why this was generated.